TIG/TIG/Extension/DateComponents+Ext.swift

```diff
@@ -33,6 +33,20 @@ extension DateComponents {
             return formattedTime + " " + minutes + "분"
         }
     }
+    
+    func formattedFullDuration() -> String {
+        var components: [String] = []
+        
+        if let hours = self.hour, hours >= 0 {
+            components.append("\(hours)시간")
+        }
+        
+        if let minutes = self.minute, minutes >= 0 {
+            components.append("\(minutes)분")
+        }
+        
+        return components.joined(separator: " ")
+    }
```

Comment:
```suggestion
    
    func formattedFullDuration() -> String {
        guard let hour = self.hour,
                   let minute = self.minute else {
                           return "0시간 00분"
                   }
    }
    
    return "\(hour)시간 \(minute)분"
```

배열을 만들고 문자열을 결합하는 방식이 아닌 문자열 리터럴 값을 바로 리턴해주는 방법이 더 간단해보입니다.
joined 함수를 사용하지 않으니.

Reply:
수정하였습니다^^